packages/cli/src/commands/preview-docs/preview-server/preview-server.ts

```diff
@@ -36,6 +37,11 @@ function getPageHTML(
       ? 'https://cdn.jsdelivr.net/npm/@redocly/reference-docs@latest/dist/redocly-reference-docs.min.js'
       : 'https://cdn.jsdelivr.net/npm/redoc@latest/bundles/redoc.standalone.js'
   }"></script>
+  ${
```

Comment:
better to use a separate page template: https://cdn.jsdelivr.net/npm/@redocly/reference-docs@latest/dist/oauth2-redirect.html

Reply:
I know, but in that case we need to host it somewhere. Is it ok ?

Reply:
done